tutor/controllers/course.py

```diff
@@ -11,6 +20,16 @@ def getCourseName(id):
     pass
 
 
+def showCourses():
+    # get courses from data base into a list
```

Comment:
I think this kind of lookup table belongs in the model code rather then in the controller code.

Reply:
we still don't have a model for that. Once DB is available it will be relocated.